src/routes/+page.ts

```diff
@@ -0,0 +1,123 @@
+import type { PageLoad } from "./$types";
+import { BackendController } from "$lib/controller/backend-controller";
+import {
+    type Paper,
+    type Project,
+    type ProjectMetadata,
+    type StageEntry,
+    type User,
+} from "$lib/model/backend";
+import { calculateStageProgress } from "$lib/utils/statistics-helper";
+
+// import type { Project, ProjectMetadata } from "$lib/model/backend";
+
+async function requestProjectMetadata(project: Project): Promise<ProjectMetadata> {
+    const members: User[] = await BackendController.getInstance().project(project.id).getMembers();
+    const currentStage: number = await BackendController.getInstance()
+        .project(project.id)
+        .getStageCount();
+    const allPapersInCurrentStage: StageEntry[] = await BackendController.getInstance()
+        .project(project.id)
+        .stage(currentStage)
+        .getPapers();
+
+    return {
+        project: project,
+        members: members,
+        stage: currentStage,
+        stageProgress: calculateStageProgress(
+            allPapersInCurrentStage.map((stageEntry: StageEntry): Paper => stageEntry.paper),
+        ),
+    };
+}
+
+/**
+ * Loads projects for the user logged in.
+ *
+ * Therefore, request the project ids of the projects, the user logged in is member of and use
+ * these ids to request:
+ * <ul>
+ *     <li>the project members</li>
+ *     <li>the current project stage</li>
+ *     <li>the progress of the current stage</li>
+ * </ul>
```

Comment:
This HTML doesn't get rendered in the VSCode tooltip. Wouldn't it just work with Markdown lists?
![image](https://github.com/user-attachments/assets/dd1550ba-fb54-4912-89f1-a68aa1e2a7c7)

```suggestion
 * - the project members
 * - the current project stage
 * - the progress of the current stage
```
which results in
![image](https://github.com/user-attachments/assets/7e2e2857-18e8-4c5c-a816-79e5a85e03fb)

Reply:
Yes, I mixed it up with the documentation style in C++